docs/user_guide/gui/recon_window.rst

```diff
@@ -16,3 +16,15 @@ COR and Tilt
 
 Is described in the section :ref:`Center of Rotation`
 
+Beam Hardening Correction
+-------------------------
+
+Mantid Imaging implements a linearization correction to compensate for the beam hardening effect. This is a applied to the projection pixel values before being passed to the reconstruction algorithm. First we convert from transmission to line density as normal:
```

Comment:
Just a small typo, but otherwise looks good. The maths seems to be rendering well.

`This is a applied to the projection pixel` -> `This is applied to the projection pixel`